go.mod

```diff
@@ -4,5 +4,8 @@ go 1.19
 
 require (
 	github.com/dblohm7/wingoes v0.0.0-20230131214643-2b26ab7fb5f9
-	golang.org/x/sys v0.4.0
+	golang.org/x/sys v0.7.0
+	golang.org/x/tools v0.8.0
 )
+
+require golang.org/x/mod v0.10.0 // indirect
```

Comment:
I think this should be combined in the require block above?

Reply:
That's what `go mod tidy` did... 🤷‍♂️